src/main/java/teammates/ui/controller/InstructorCourseStudentDetailsEditSaveAction.java

```diff
@@ -67,11 +70,21 @@ public ActionResult execute() throws EntityDoesNotExistException {
             
             logic.updateStudent(studentEmail, student);
             
+            boolean isSessionSummarySendEmail = getRequestParamAsBoolean(Const.ParamsNames.SESSION_SUMMARY_EMAIL_SEND_CHECK);
             if (isEmailChanged) {
```

Comment:
Something is wrong here. It looks like the value `isSessionSummarySendEmail` is not used?